events/imperia_frankfurt_parliament_events.txt

```diff
@@ -0,0 +1,1105 @@
+		# NGF cannot have spawned at this point
+		if = {
+			limit = {
+				is_target_in_global_variable_list = {
+					name = german_establishment_aligned
+					target = c:PRU
+				}
+			}
+			# Slight sidetrack to puppet Prussia, they dont get caught by the above loop for some reason...
+			root = {
+				create_diplomatic_pact = {
+					country = c:PRU
+					type = dominion
+				}
+			}
+			imperia_frankfurt_parliament_spawn_rhineland = yes
+			root = {
+				create_diplomatic_pact = {
+					country = c:RHE
+					type = puppet
+				}
+			}
+		}
+		imperia_frankfurt_parliament_consolidate_gutter_king = {
+			GUTTER_KING = root
+		}
+	}
+}
+
+# Hidden event for starting the civil war
+frankfurt_parliament.9 = {
+	type = country_event
+	hidden = yes
+	trigger = {
+		any_country = {
+			has_journal_entry = je_imperia_frankfurt_german_consolidation_german_major
+		}
+	}
+	immediate = {
+		every_country = {
+			limit = {
+				OR = {
+					has_journal_entry = je_imperia_frankfurt_german_consolidation_gutter_king
+					has_journal_entry = je_imperia_frankfurt_german_consolidation_german_major
+					has_journal_entry = je_imperia_frankfurt_german_consolidation_german_minor
+				}
+			}
+			if = {
+				limit = {
+					has_journal_entry = je_imperia_frankfurt_german_consolidation_gutter_king
+				}
+				save_scope_as = gutter_king
+			}
+			else_if = {
+				limit = {
+					has_journal_entry = je_imperia_frankfurt_german_consolidation_german_major
+				}
+				add_to_global_variable_list = {
+					name = german_establishment_aligned
+					target = this
+				}
+			}
+			else_if = {
+				limit = {
+					has_journal_entry = je_imperia_frankfurt_german_consolidation_german_minor
+				}
+				if = {
+					limit = {
+						OR = {
+							has_modifier = gutter_king_solid_sway
+							has_modifier = gutter_king_complete_sway
+						}
+					}
+					add_to_global_variable_list = {
+						name = gutter_king_aligned
+						target = this
+					}
+				}
+				else_if = {
+					limit = {
+						OR = {
+							has_modifier = german_major_solid_sway
+							has_modifier = german_major_complete_sway
+						}
+					}
+					add_to_global_variable_list = {
+						name = german_establishment_aligned
+						target = this
+					}
+				}
+			}
+		}
+		ordered_in_global_list = {
+			variable = german_establishment_aligned
+			order_by = gdp
+			position = 0
+			save_scope_as = german_major
+			set_variable = german_establishment_leader
+		}
+		scope:german_major = {
+			create_diplomatic_play = {
+				type = dp_imperia_gutter_german_revolt
+				target_country = scope:gutter_king
+			}
+		}
+		random_diplomatic_play = {
+			limit = {
+				initiator_is = scope:german_major
+				target_is = scope:gutter_king
+			}
+			set_variable = gutter_german_revolt_play
+		}
+		every_in_global_list = {
+			variable = gutter_king_aligned
+			random_diplomatic_play = {
+				limit = {
+					has_variable = gutter_german_revolt_play
```

Comment:
Can't you just use is_diplomatic_play_type?

Reply:
Doh, yes I can. The variable bit is from when I first started on this and hadn't made a custom dp_type yet.
I'll take a look to see if that variable is needed for anything else.